cmd/util/ledger/reporters/fungible_token_tracker.go

```diff
@@ -151,7 +152,7 @@ func (r *FungibleTokenTracker) worker(
 
 		for _, domain := range domains {
 			storageMap := storage.GetStorageMap(owner, domain)
-			itr := storageMap.Iterator()
+			itr := storageMap.Iterator(&interpreter.Interpreter{})
```

Comment:
@SupunS TestOrderedMapMigration is now failing (after upgrading to the latest commit of the memory-metering branch) and I think it relates to this change, but I am not sure how to fix it.

Any Ideas?

```
--- FAIL: TestOrderedMapMigration (0.00s)
    --- FAIL: TestOrderedMapMigration/sort_values (0.00s)
panic: cannot convert stored value: migrations.RawStorable [recovered]
	panic: cannot convert stored value: migrations.RawStorable

goroutine 10 [running]:
testing.tRunner.func1.2(0x4add640, 0xc0002c2740)
	/usr/local/go/src/testing/testing.go:1143 +0x332
testing.tRunner.func1(0xc000102d80)
	/usr/local/go/src/testing/testing.go:1146 +0x4b6
panic(0x4add640, 0xc0002c2740)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
github.com/onflow/cadence/runtime/interpreter.MustConvertStoredValue(...)
	/Users/janez/go/pkg/mod/github.com/onflow/cadence@v0.21.3-0.20220316204141-2e995e2195c8/runtime/interpreter/storage.go:48
github.com/onflow/cadence/runtime/interpreter.StoredValue(0x4e55340, 0xc000102c00, 0x4e69740, 0xc000556150, 0x4e6d030, 0xc000560e10, 0xc000556168, 0x4e69740)
	/Users/janez/go/pkg/mod/github.com/onflow/cadence@v0.21.3-0.20220316204141-2e995e2195c8/runtime/interpreter/storage.go:42 +0xdf
github.com/onflow/cadence/runtime/interpreter.StorageMap.SetValue(0xc000561380, 0xc000102c00, 0xc000482af7, 0x3, 0x4e58360, 0xc000556168)
	/Users/janez/go/pkg/mod/github.com/onflow/cadence@v0.21.3-0.20220316204141-2e995e2195c8/runtime/interpreter/storagemap.go:127 +0x194
github.com/onflow/flow-go/cmd/util/ledger/migrations.(*OrderedMapMigration).migrate(0xc000506090, 0xc000427e80, 0x5, 0x5, 0x40, 0x10, 0xc000427dd0, 0x2, 0x2)
	/Users/janez/Programing/dl/flow-go/cmd/util/ledger/migrations/ordered_map_migration.go:237 +0x41c
github.com/onflow/flow-go/cmd/util/ledger/migrations.TestOrderedMapMigration.func2(0xc000102d80)
	/Users/janez/Programing/dl/flow-go/cmd/util/ledger/migrations/ordered_map_migration_test.go:88 +0xb4c
testing.tRunner(0xc000102d80, 0xc000561050)
	/usr/local/go/src/testing/testing.go:1193 +0xe5
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1238 +0x2b3
```

Reply:
Just saw this. Having a look now 👀

Reply:
It was due to an API change in `interpreter.Value` interface. Pushed a fix: 8c597980e5956cd90c5b7aa90b2d94e1ef9ce505